open-metadata-test/open-metadata-fvt/access-services-fvt/governance-program-fvt/build.gradle

```diff
@@ -0,0 +1,110 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright Contributors to the ODPi Egeria project.
+ */
+
+plugins {
+    // Include plug-in for starting a new process
+    id 'com.github.hesch.execfork'
```

Comment:
need to follow changes as per https://github.com/odpi/egeria/commit/4834f05815ac439b5dd3b49ddb442dd8c9ccbc10